src/ui_ng/src/app/base/start-page/start.component.html

```diff
@@ -19,7 +19,7 @@ <h3 class="card-title">Getting Start</h3>
                 </p>
             </div>
             <div class="card-footer my-card-footer">
-                <a href="http://vmware.github.io/harbor/" target="_blank" class="btn btn-sm btn-link">Learn More</a>
+                <a href="https://github.com/goharbor/harbor/blob/master/README.md" target="_blank" class="btn btn-sm btn-link">Learn More</a>
```

Comment:
This should be `https://goharbor.io/harbor/` ?

Reply:
Actually, just https://goharbor.io is fine.

Reply:
done.